assets/helpers/abTests/abtest.js

```diff
@@ -100,6 +121,30 @@ function getParticipationsFromUrl(): ?Participations {
   return null;
 }
 
+function userInBreakpoint(audience: Audience): boolean {
+
+  if (!audience.breakpoint) {
+    return true;
+  }
+
+  const { minWidth } = audience.breakpoint;
+  const minWidthMediaQuery = minWidth ? `(min-width:${breakpoints[minWidth]}px)` : null;
+
+  const { maxWidth } = audience.breakpoint;
+  const maxWidthMediaQuery = maxWidth ? `(max-width:${breakpoints[maxWidth]}px)` : null;
+
+  if (!(minWidth || maxWidth)) {
+    return true;
+  }
```

Comment:
If you were to do:

```jsx
const { minWidth, maxWidth } = audience.breakpoint;

if (!(minWidth || maxWidth)) {
  return true;
}

const minWidthMediaQuery = minWidth ? `(min-width:${breakpoints[minWidth]}px)` : null;
const maxWidthMediaQuery = maxWidth ? `(max-width:${breakpoints[maxWidth]}px)` : null;
```

you could return earlier in the case where they're both empty, and avoiding running the code that generates the template strings?

Reply:
👍 Good call